CMakeLists.txt

```diff
@@ -32,11 +32,10 @@
 # 3. cmake ..
 # 4. make -j
 
-cmake_minimum_required(VERSION 2.8.12)
-project(rocksdb)
-enable_language(CXX)
-enable_language(C)
-enable_language(ASM)
+cmake_minimum_required(VERSION 3.5.1)
+project(rocksdb
+  VERSION 6.2.0
```

Comment:
is it possible to get version from version.h?

Reply:
it's better to not duplicate in two places. Is "VERSION 6.2.0" needed here?

Reply:
agreed. i extracted `ReadVersion.cmake` out, and use it instead.